src/main/java/circle/Cylinder.java

```diff
@@ -0,0 +1,51 @@
+package circle;
+
+public class Cylinder extends Circle {
+    private double height;
+    final double PI = 3.14;
```

Comment:
This constant is already defined in the parent class as package-local so redefining is redundant. And you can use Math.PI instead

Reply:
Fixed.